configs/components/boost.rb

```diff
@@ -0,0 +1,183 @@
+component "boost" do |pkg, settings, platform|
```

Comment:
Why not use system packages instead of building our own? How does this improve things of pl-boost?